.env.local

```diff
@@ -0,0 +1,26 @@
+ROOT_LOGGING_LEVEL=info
+APP_LOGGING_LEVEL=info
+WEB_LOGGING_LEVEL=info
+
+MONGO_HOST="localhost"
```

Comment:
Maybe is no problem, but sometimes there is a connectivity error when using localhost instead of container explicit reference, consider changing this to:
MONGO_HOST="mongodb"
MONGO_PORT=27017  <-- the internal docker port